src/sentry/lang/native/dsymcache.py

```diff
@@ -24,19 +24,14 @@ def dsym_cache_path(self):
     def get_project_path(self, project):
         return os.path.join(self.dsym_cache_path, six.text_type(project.id))
 
-    def get_global_path(self):
```

Comment:
All the changes here just remove the system dsym files. We use the symbolserver now.